httpx/auth.py

```diff
@@ -104,6 +104,8 @@ def __init__(
         self.password = to_bytes(password)
 
     def __call__(self, request: Request) -> AuthFlow:
```

Comment:
Wasn't the issue specific to `DigestAuth` ?

Reply:
I guess it is. Can you be more specific what the issue?

Reply:
Yeah, my bad, I thought the changes were in Base class